.github/workflows/automerge-transifex-app-prs.yml

```diff
@@ -15,26 +15,15 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: merge pull request
-        uses: nick-fields/retry@v2
-        id: mergePR
+
+      - name: Auto-merge pull request
         env:
           # secrets can't be used in job conditionals, so we set them to env here
           TRANSIFEX_APP_ACTOR_NAME: "${{ secrets.TRANSIFEX_APP_ACTOR_NAME }}"
           TRANSIFEX_APP_ACTOR_ID: "${{ secrets.TRANSIFEX_APP_ACTOR_ID }}"
           # This token requires Write access to the openedx-translations repo
           GITHUB_TOKEN: ${{ secrets.EDX_TRANSIFEX_BOT_GITHUB_TOKEN }}
-          PR_NUMBER: ${{ github.event.number }}
         if: "${{ github.actor == env.TRANSIFEX_APP_ACTOR_NAME && github.actor_id == env.TRANSIFEX_APP_ACTOR_ID }}"
-        with:
-          retry_wait_seconds: 60
-          max_attempts: 5
-          timeout_minutes: 15
-          retry_on: error
-          command: |
-            # Attempt to merge the PR
-            gh pr merge ${{ github.head_ref }} --rebase --auto
-
-            # The `fromdate | todate` are used merge to validate that `mergedAt` isn't null
-            # therefore verifying that the pull request was merged successfully.
-            gh pr view "$PR_NUMBER" --json mergedAt --jq '.mergedAt | fromdate | todate'
+        run: |
+          # Add the pull request to the merge queue with --rebase commit strategy
+          gh pr merge ${{ github.head_ref }} --rebase --auto
```

Comment:
This is the same original `merge --rebase --auto` but now within a GitHub Actions native `run` instead of `retry`.